control_plane/src/local_env.rs

```diff
@@ -23,7 +23,7 @@ use crate::object_storage::{OBJECT_STORAGE_REMOTE_STORAGE_DIR, ObjectStorage};
 use crate::pageserver::{PAGESERVER_REMOTE_STORAGE_DIR, PageServerNode};
 use crate::safekeeper::SafekeeperNode;
 
-pub const DEFAULT_PG_VERSION: u32 = 16;
+pub const DEFAULT_PG_VERSION: u32 = 17;
```

Comment:
We have a couple of other places where we define `DEFAULT_PG_VERSION` in rust code:
```
control_plane/src/local_env.rs:pub const DEFAULT_PG_VERSION: u32 = 16;
pageserver/src/lib.rs:pub const DEFAULT_PG_VERSION: u32 = 16;
```

Should we update them as well?

We probably should update it in Python as well (ok to keep CI on v16 for now), 
see the previous update to `v16` https://github.com/neondatabase/neon/pull/8745

Reply:
I'll go ahead and do them all and update the PR title/commit message